config/index.js

```diff
@@ -762,6 +768,12 @@ var conf = convict({
       format: 'duration',
       env: 'SECONDARY_EMAIL_MIN_UNVERIFIED_ACCOUNT_TIME'
     }
+  },
+  signinCodeSize: {
```

Comment:
I thought about that but `signinCodeSize` really has nothing to do with the SMS stuff at all and `signinCodesBaseUri` really does.

Maybe I should name one or other of them differently because I agree that it seems weird at first glance they're not next to each other at the same level of the config. Having a think...